src/modules/notifications/components/notifications-view.styles.less

```diff
@@ -2,18 +2,18 @@
 
 .NotificationsView {
   background: @color-lightergray;
-  box-shadow: 0.1rem 0.2rem 0.5rem @color-almostblack;
+  box-shadow: @box-shadow-notifications;
   color: @color-text-dark;
-  display: block;
+  display: flex;
   height: 30.25rem;
   max-height: 80%;
   padding: 0.25em;
   position: absolute;
-  right: 0.25rem;
+  right: 1.8rem;
   text-align: left;
   top: 3rem;
-  width: 20em;
-  z-index: 1000;
+  width: 22.75rem;
+  z-index: @mask-modal;
```

Comment:
Can you confirm that this won’t cause the modal to reside beneath the notifications when present?

@mask-modal is intended for full view modals only.

Reply:
switched this to mask-above. it's still mask-modal on mobile however as it is intended to act as a full screen display. Is this going to need it's own z-index instead so that we can show a overall page modal?